frontend/src/components/common/Filter/common/constants.ts

```diff
@@ -1,4 +1,1121 @@
-/* Case insensitive sorter */
+import { EVENTS } from "src/common/analytics/events";
```

Comment:
The functionality in this file was moved *mostly* unchanged from useCategoryFilter. I have marked the changes below.